src_assets/common/assets/web/troubleshooting.html

```diff
@@ -82,23 +82,38 @@ <h2 id="restart">Restart Sunshine</h2>
       </div>
     </div>
     <!-- Unpair all Clients -->
-    <div class="card p-2 my-4">
+    <div class="card my-4">
       <div class="card-body">
-        <h2 id="unpair">Unpair All Clients</h2>
-        <br>
-        <p>Remove all your paired devices</p>
-        <div class="alert alert-success" v-if="unpairAllStatus === true">
-          Unpair Successful!
-        </div>
-        <div class="alert alert-danger" v-if="unpairAllStatus === false">
-          Error while unpairing
-        </div>
-        <div>
-          <button class="btn btn-danger" :disabled="unpairAllPressed" @click="unpairAll">
-            Unpair All
-          </button>
+        <div class="p-2">
+          <div class="d-flex justify-content-end align-items-center">
+            <h2 class="text-center me-auto">Unpair Clients</h2>
+            <button class="btn btn-danger" :disabled="unpairAllPressed" @click="unpairAll">
+              Unpair All
+            </button>
+          </div>
+          <div id="apply-alert" class="alert alert-success d-flex align-items-center mt-3" :style="{ 'display': (showApplyMessage ? 'flex !important': 'none !important') }">
+            <div class="me-2"><b>Success!</b> Click 'Apply' to restart Sunshine and apply changes. This will terminate any running sessions.</div>
```

Comment:
Is it necessary to restart Sunshine? I'm guessing you're doing this to force disconnecting a client if it was one of those that were removed?

There's probably a way we can disconnect that single client instead of restarting Sunshine altogether.

Reply:
Two reasons:
1. Yes, to disconnect the client.
2. The certificate is unfortunately is still valid after unpairing a single client, so it can simply reconnect without issue. I'm not sure why. If someone can fix this, then only the session would need to end.

If access can be fully revoked and the client is disconnected, I think that would be a lot better than requesting a restart.

Reply:
@cgutman do you what we can do to remove a single client without restarting sunshine?

Reply:
I think this is the only remaining concern.

Reply:
I think we would have to remove it from `cert_chain`. In https://github.com/LizardByte/Sunshine/pull/2365/files I am removing all of them for the unpair all.